src/js/constants/DefaultErrorMessages.js

```diff
@@ -0,0 +1,164 @@
+const DefaultErrorMessages = [
+
+  //
+  // The following rule will match every message path, behaving
+  // like the default sink if a path is not found.
+  //
+  // Remember to concat these error messages when you are composing
```

Comment:
Will good old inheritance solve the remembering issue?
Yes, it's not gonna be a plain array, but one less thing to remember